content/akeneo-event-platform/best-practices.md

```diff
@@ -0,0 +1,29 @@
+# Best practices
+
+## Queuing Events
+
+To maintain performance with a high volume of subscriptions, the event platform requires a response within the specified time limit, please refer to [this page](/akeneo-event-platform/concepts.html#ack-timeout) to know more. Handle received events using a queue or asynchronously to ensure you acknowledge the event quickly, before processing it.
+
+## Handling 429 Responses
+
+The platform can send a large number of events in a short time, potentially causing overload and leading to `500` responses from your side. In such cases, your subscription may be suspended. By implementing a proper `429` response mechanism, you can delay event processing to maintain service stability and avoid suspension.
```

Comment:
I'm confused here :) 

The platform will not "handle 429 responses" meaning: we will not take care of slowing down or wait the "specified amount of time" to try to deliver the event again.

For https destination, I think we should emphasize the idea of "respond 200 quickly", yes our platform will try to publish event as fast as it can which can be challenging for an http endpoint to handle. We will suspend your subscription if your endpoint answer anything than 200, or revoke for 404. To ensure to handle a high pace event traffic, prefer to use a PubSub destination.

Reply:
i'll wait for @fchoquet before changing this